cob_description/urdf/sensors/sick_s300_laser.gazebo.xacro

```diff
@@ -13,8 +13,8 @@
             <horizontal>
               <samples>541</samples>
               <resolution>1.0</resolution>
-              <min_angle>${min_angle}</min_angle>
```

Comment:
I do not agree:
The simulated sensor should reflect the real hardware specs and not robot-specifics....thus, for cropping the laser scans, the laser_scan_filter should be used....with the values you mentioned above...